### ISSUE TITLE
Question about padding side at RL model initialization.

### ISSUE THREAD

**Issue by L4zyy** [llava-rlhf/LLaVA-RLHF#29]
Hi, thank you for sharing this great work.

When went through your RLHF training code, I noticed that the padding approach in the RL model initialization step differs from the other two steps, as it utilizes right padding. Could you kindly clarify the rationale behind using right padding?

https://github.com/llava-rlhf/LLaVA-RLHF/blob/830a083fd83e607da3c35f2e6aef833523c1e722/RLHF/finetune_lora_sft_ds.py#L398

**Comment by Edward-Sun** [comment]
Hi @L4zyy 

The "right" padding in SFT and RL initialization is because it doesn't matter whether to use "left" or "right" padding. So we just follow the common practice such as in LLaVA to use the "left" padding.

The "left" padding in RL and reward modeling is because RL needs to collect policy rollouts, while reward modeling needs to do a regression on the last token, both of which cases require the model to aligned on the right side (new token side).
